model/BUILD.bazel

```diff
@@ -149,7 +149,8 @@ proto_library(
 
 java_lite_proto_library(
     name = "voiceover_java_proto_lite",
-    deps = [":voiceover_proto"],
+    visibility = ["//visibility:public"],
+    deps = [":voiceover_proto"],    
```

Comment:
there is extra space at the end of the line. Removing it will solve the linter issue.

Reply:
Yes, that's it, nice catch.

Reply:
Addressed this in the a new commit

Reply:
@anandwana001 @fsharpasharp  Even after removing the whitespace and using ```./buildifier --lint=warn --mode=check --warnings=-native-android,+out-of-order-load,+unsorted-dict-items model/BUILD.bazel  ``` The lint check doesn't seem to pass.

Also, there seems to be some issue in the app level BUILD.bazel file as ```bazel test //app/...``` still fails build
![image](https://user-images.githubusercontent.com/55680995/112309600-87d30200-8cc9-11eb-89fb-c15a6bdc22ea.png)